pagopa/pn-frontend
# notiifcation detail - aar copy - pn-5873

## Short description
Changed the copies of the title and subtitle of the old section "Altri documenti"

## List of changes proposed in this pull request
- Updated localization files (PG/PA/PF)
- Updated NotificationDetail components (PG/PA/PF)

## How to test
Login with PG/PA/PF -> go to notification detail -> check that the copies of the old section "Altri documenti" are the same of the linked issue (check the comments)
